src/main/java/software/bernie/techarium/Techarium.java

```diff
@@ -1,8 +1,10 @@
 package software.bernie.techarium;
 
+import blusunrize.immersiveengineering.client.DynamicModelLoader;
```

Comment:
Remove this import. It's unused as far as i see and stops the mod from working, if immersive is not present
```suggestion

```